include/arch/arm/aarch32/cortex_m/fpu.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _CORTEX_M_FPU_H
```

Comment:
Names starting with `_` is deprecated.
```suggestion
#ifndef ZEPHYR_INCLUDE_ARCH_ARM_AARCH32_CORTEX_M_FPU_H_
```

Reply:
@oyvindronningstad ^^

Reply:
done